source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -92,30 +92,36 @@ RasterYuv420Semiplanar: class extends RasterImage {
 		}
 		result
 	}
-	resizeInto: func (target: This) {
+	resizeInto: func (target: This, sourceBox := IntBox2D new()) {
+		if (sourceBox hasZeroArea)
+			sourceBox = IntBox2D new(this size)
```

Comment:
I suppose `sourceBox := IntBox2D new(this size)` does not work as a default argument?